tests/migration/online_migration/pre_migration.pm

```diff
@@ -48,6 +48,9 @@ sub remove_kgraft_patch {
 sub run {
     select_console 'root-console';
 
+    # set_zypp_single_rpmtrans
+    # if libzypp-17.28.4 and zypper-1.14.49 or higher version installed
```

Comment:
I guess comments in 52 and 51 are not needed anymore

Reply:
done.